pkgs/data/themes/adw-gtk3/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, nix-update-script
+, meson
+, ninja
+, sassc
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "adw-gtk3";
+  version = "3.7";
+
+  src = fetchFromGitHub {
+    owner = "lassekongo83";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hHmNRPUJOXa//aKgAYhGBVX6usRsObWbzcfOa1uwbqM=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    sassc
+  ];
+
+  postPatch = ''
+    chmod +x gtk/src/adw-gtk3-dark/gtk-3.0/install-dark-theme.sh
+    patchShebangs gtk/src/adw-gtk3-dark/gtk-3.0/install-dark-theme.sh
+  '';
+
+  passthru = {
+    updateScript = nix-update-script {
+      attrPath = pname;
+    };
+  };
+
+  meta = with lib; {
+    description = "The theme from libadwaita ported to GTK-3";
+    homepage = "https://github.com/lassekongo83/adw-gtk3";
+    license = licenses.lgpl21Only;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ "ciferkey" ];
```

Please add yourself to `maintainers/maintainer-list.nix` in a separate commit `maintainers: ciferkey` and change this to:

```suggestion
    maintainers = with maintainers; [ ciferkey ];
```

---

Thanks for the pointer!